templates/workloads/jobs/scheduled-job/manifest.yml

```diff
@@ -10,7 +10,7 @@ type: {{.Type}}
 
 image:
 {{- if .ImageConfig.Build.BuildArgs.Dockerfile}}
-  # Docker build arguments. You can specify additional overrides here. Supported: dockerfile, context, args.
+  # Docker build arguments. You can specify additional overrides here. Supported: dockerfile, context, args, target, cache_from.
```

Comment:
What do you think of just linking to the website here instead (same for the other manifest files):
```suggestion
  # Docker build arguments. 
  # For additional overrides: https://aws.github.io/copilot-cli/docs/manifest/scheduled-job/#image-build
```

Reply:
Done, good idea :+1: